msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -70,6 +76,19 @@
  * &lt;/activity&gt;
  * </pre>
  * </p>
+ * <p>Other Terminology:</p>
+ * <p>
+ *    <p><b>Scopes:</b>Permissions that the developers wants included in the access token received . Not all scopes are
+ *     guaranteed to be included in the access token returned.
```

Comment:
nit: Is there a way by which the user can check what scopes were consented to? If yes, we should atleast say here, look at documentation for **api to check which scopes were consented to, if not ignore this comment